install_magenta.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# Check which operating system
+if [ "$(uname)" == "Darwin" ]; then
```

Comment:
[[ is preferred over [
https://google.github.io/styleguide/shell.xml#Test,_%5B_and_%5B%5B